### ISSUE TITLE
Trouble loading Albert model

### ISSUE THREAD

**Issue by jswift24** [huggingface/transformers#2228]
## 🐛 Bug

Model I am using (Bert, XLNet....): Albert

Language I am using the model on (English, Chinese....): English

The problem arise when using:
* [x] the official example scripts: (give details):
* [ ] my own modified scripts: (give details)

The tasks I am working on is:
* [x] an official GLUE/SQUaD task: (give the name) 
* [ ] my own task or dataset: (give details)

Using example from docs: https://huggingface.co/transformers/model_doc/albert.html

## To Reproduce
Trying to load the Albert model using the code below:
```
import tensorflow as tf
from transformers import AlbertTokenizer
tokenizer = AlbertTokenizer.from_pretrained('bert-base-uncased')
```
Getting the following error:
```
Traceback (most recent call last):

  File "<ipython-input-4-56254e5f4b51>", line 3, in <module>
    tokenizer = AlbertTokenizer.from_pretrained('bert-base-uncased')

  File "c:\users\admin\appdata\local\programs\python\python37\lib\site-packages\transformers\tokenization_utils.py", line 302, in from_pretrained
    return cls._from_pretrained(*inputs, **kwargs)

  File "c:\users\admin\appdata\local\programs\python\python37\lib\site-packages\transformers\tokenization_utils.py", line 437, in _from_pretrained
    tokenizer = cls(*init_inputs, **init_kwargs)

  File "c:\users\admin\appdata\local\programs\python\python37\lib\site-packages\transformers\tokenization_albert.py", line 90, in __init__
    self.sp_model.Load(vocab_file)

  File "c:\users\admin\appdata\local\programs\python\python37\lib\site-packages\sentencepiece.py", line 118, in Load
    return _sentencepiece.SentencePieceProcessor_Load(self, filename)

RuntimeError: Internal: C:\projects\sentencepiece\src\sentencepiece_processor.cc(73) [model_proto->ParseFromArray(serialized.data(), serialized.size())] 
```

Note, the same code works for bert instead of albert:
> from transformers import *
> tokenizer = BertTokenizer.from_pretrained('bert-base-cased')
> 

## Expected behavior
Looking for albert model to load without errors

## Environment

* OS: Windows 10
* Python version: 3.7
* PyTorch version: 1.3.1
* PyTorch Transformers version (or branch): pip install transformers (fresh install today 12/19/2019)
* Using GPU ?: Yes
* Distributed of parallel setup ?: N/A
* Any other relevant information:

**Comment by LysandreJik** [comment]
Indeed, that's an error from the docs! I just fixed it with 33adab2. The doc should be updated now, thanks for raising this issue.

**Comment by jswift24** [comment]
Wow, that was fast. Thank you!
